tests/mol-audio_no_clipbegin/OPS/content_001.xhtml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<html xmlns="http://www.w3.org/1999/xhtml">
+	<head>
+        <meta charset="utf-8" />
+		<title>Automatic start at the beginning of an audio track</title>
+	</head>
+	<body>
+
+        <p>Test passes (i.e., the Reading System correctly implements the lack of a <code>clipBegin</code> attribute value) if</p>
+
+        <ol>
+            <li>The Reading System provides the user with a way to turn text reading on</li>
+            <li>If turned on, the extract on the next page is read aloud by the Reading System</li>
+            <li>The text does <em>not</em> start at the beginning of the text but at the title of the publication, name of the reader, etc.</li>
```

Comment:
Should "The text does not start" be "The audio does not start" or similar? Not a Media Overlays expert but "text" seems to me like the wrong word.